src/commands/key_command.rs

```diff
@@ -345,6 +347,13 @@ impl std::str::FromStr for KeyCommand {
                     format!("{}: {}", command, e.to_string()),
                 )),
             },
+            "tab_switch_index" => match arg.parse::<i32>() {
```

Comment:
This can be changed to `usize` or `u32` since index is always going to be positive